playbook/app/pb_kits/playbook/pb_fixed_confirmation_toast/fixed_confirmation_toast.rb

```diff
@@ -60,7 +60,10 @@ def icon_class
       end
 
       def classname
-        generate_classname("pb_fixed_confirmation_toast_kit", status, multi_line_class) + close_class + position_class + auto_close_class + icon_class
+        default_z_index = z_index.present? ? "" : " z_index_max"
+        # IMPORTANT: the AutoClose class must be the last class in the string for JS to read it correctly
+        # Changing the order will break the auto_close functionality
+        generate_classname("pb_fixed_confirmation_toast_kit", status, multi_line_class) + close_class + position_class + icon_class + default_z_index + auto_close_class
```

Comment:
Fun fact, the auto_close_class needs to be the last class for this to work, which is not ideal. For now added the comment since its been this way since the prop was created, but we should look into changing the enhanced element logic to look for a data attribute instead of reading the classname. The current code is very brittle.